### PR TITLE
config: params modified to use configuration from database

### PR DESCRIPTION
**Pull request by kadhikari**
https://jira.kisio.org/browse/NAVP-1439

### CODE REVIEW

**Comment by mbounabi** [comment]
@pbougue isn't this supposed to remove only default values? If the env variables are still set for any platform, it will work

File: kirin/default_settings.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 # COTS configuration
 # * external configuration
-COTS_CONTRIBUTOR = os.getenv("KIRIN_COTS_CONTRIBUTOR", "realtime.cots")
+COTS_CONTRIBUTOR = os.getenv("KIRIN_COTS_CONTRIBUTOR", None)
```
Comment (pbougue) [inline kirin/default_settings.py]: This is not defined in fab_kirin so it will break

File: kirin/default_settings.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 NAVITIA_TIMEOUT = int(os.getenv("KIRIN_NAVITIA_TIMEOUT", 5))
 
-NAVITIA_INSTANCE = os.getenv("KIRIN_NAVITIA_INSTANCE", "sncf")
+NAVITIA_INSTANCE = os.getenv("KIRIN_NAVITIA_INSTANCE", None)
```
Comment (pbougue) [inline kirin/default_settings.py]: If we change kirin_deployment_conf: this one is defined twice in common.py, we should remove one.

File: kirin/default_settings.py [hunk 3]
```diff
@@ -55,9 +55,9 @@
 
 
 # TODO : Remove when conf from db is ready
-NAVITIA_GTFS_RT_INSTANCE = os.getenv("KIRIN_NAVITIA_GTFS_RT_INSTANCE", "sherbrooke")
+NAVITIA_GTFS_RT_INSTANCE = os.getenv("KIRIN_NAVITIA_GTFS_RT_INSTANCE", None)
 NAVITIA_GTFS_RT_TOKEN = os.getenv("KIRIN_NAVITIA_GTFS_RT_TOKEN", None)
-GTFS_RT_CONTRIBUTOR = os.getenv("KIRIN_GTFS_RT_CONTRIBUTOR", "realtime.sherbrooke")
+GTFS_RT_CONTRIBUTOR = os.getenv("KIRIN_GTFS_RT_CONTRIBUTOR", None)
```
Comment (pbougue) [inline kirin/default_settings.py]: This one is defined only in every config file in kirin_deployment_conf, better to have it in common.py

**Comment by kadhikari** [comment]
If we have configuration in kirin_deployment_conf for all those parameters with None in kirin, this PR should work

File: kirin/default_settings.py (hunk 3 above)
Comment (kadhikari) [reply kirin/default_settings.py]: We better deactivate in each platform during migration "env.gtfs_rt_contributor = None"

File: kirin/default_settings.py (hunk 2 above)
Comment (kadhikari) [reply kirin/default_settings.py]: Done in fab_kirin: https://github.com/CanalTP/kirin_deployment_conf/pull/39/files

File: kirin/default_settings.py (hunk 1 above)
Comment (kadhikari) [reply kirin/default_settings.py]: Added in fab_kirin and kirin_deployment_conf

**Pull request merged**